pkg/cmd/issue/comment/comment_test.go

```diff
@@ -234,13 +235,29 @@ func Test_commentRun(t *testing.T) {
 				InteractiveEditSurvey: func(string) (string, error) { return "comment body", nil },
 				ConfirmSubmitSurvey:   func() (bool, error) { return true, nil },
 			},
+			emptyComments: true,
```

Comment:
While reviewing these tests, I felt it was important to try capturing the comment state more explicitly as an aid to maintainers. 🤔